docs/workflow.md

```diff
@@ -0,0 +1,372 @@
+# Workflow
+
+This document outlines the steps for testing the Loss To Follow-Up (LTFU) workflow, in addition to documenting the various endpoints available on the mediator. It provides a comprehensive guide on navigating the LTFU workflow and utilizing the endpoints to facilitate the necessary actions.
+
+## Environments
+
+The document provided includes placeholders for URLs. Replacing these placeholders with the appropriate endpoints for your specific environment is essential to utilize the document correctly. Below are the endpoints provided for each available environment. It is important to note that if your setup differs from the documentation provided, you may need to use different endpoints. By ensuring that the correct endpoints are used, you can be confident in successfully implementing and utilizing the LTFU workflow.
+
+### Docker - Local Setup
+
+- **Mediator** - http://localhost:5001/mediator
+- **OpenHIM Admin Console** - http://localhost:9000/
+- **CHT with LTFU configuration** - https://localhost:80/
```

Comment:
The correct link for me is: `http://localhost:5988/`. Port `80` doesn't work correctly.